### DISCUSSION TITLE
Subdomain based container rules

### DISCUSSION THREAD

**Discussion by tomervazana** [mozilla/multi-account-containers#2666] [Ideas]
When I'm using Firefox Multi-Account Containers with the option "Limit to designated sites" enabled, I encounter an annoying issue when navigating between different subdomains of the same website.

For example, if you navigate from "https://www.google.com" to "https://accounts.google.com," you will exit the "Google" container until that subdomain is manually added to the container.

I thought it would be nice if I could somehow create a container for "https://*.google.com/" (where * acts as a wildcard, as explained in the note below).

Note: The RegEx (or wildcard) * should only allow alphanumeric variations to prevent security vulnerabilities. For instance, "https://www.PhishingSite.com/WeStealYourAccount.google.com" could bypass the container if * is not restricted properly. In this case, * could mistakenly refer to "www.PhishingSite.com/WeStealYourAccount"

I asked if it can be done and was advised to share this idea here. [My Mozilla post](https://support.mozilla.org/en-US/questions/1463204)

If you wish to implement this and want additional details about how I would implement this, send me an email [tomer.vazana.ai@gmail.com](mailto:tomer.vazana.ai@gmail.com)

**Comment by groovecoder** [comment]
Thanks for your post. This enhancement request is filed as an issue here: https://github.com/mozilla/multi-account-containers/issues/473

Please upvote there.
